pkg/resources/cluster/patches/csi-controller-disable-volume-group-snapshots.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- target:
+    kind: Deployment
+    name: linstor-csi-controller
+  patch: |
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: linstor-csi-controller
+    spec:
+      template:
+        spec:
+          containers:
+          - name: csi-snapshotter
+            env:
+            - name: FEATURE_GATES
+              value: CSIVolumeGroupSnapshot=false
```

Comment:
If this patch is applied then it will also remove any other FEATURE_GATES that we might later enable, right? How about having separate env vars for each? E.g. `FEATURE_GATE_CSI_VOLUME_GROUP_SNAPSHOT`?